utils/api_checker/sdk-module-lists/infer-imports.py

```diff
@@ -15,6 +15,31 @@ def get_immediate_subdirectories(a_dir):
             if os.path.isdir(os.path.join(a_dir, name))]
 
 
+def collect_catalyst_frameworks(frameworks_path):
+    names = []
+    for frame in os.listdir(frameworks_path):
+        if frame.endswith(".framework"):
+            name = frame[:-len(".framework")]
+            module_dir_path = frameworks_path + '/' + frame + '/Modules'
```

Comment:
The python idiomatic way to concatenate path components is using `os.path.join`, like this: `os.path.join(frameworks_path, frame, Modules)`. You can pass any number of arguments.

Reply:
Fixed.